app/views/providers/allocations/number_of_places.html.erb

```diff
@@ -21,25 +21,26 @@
                     @provider.recruitment_cycle_year,
                   ),
                 skip_enforcing_utf8: true,
-                method: :get do |form| %>
-
-      <%= hidden_field_tag "training_provider_code", params[:training_provider_code]  %>
-
-      <fieldset class="govuk-fieldset">
-        <%= legend %>
-
-        <%= render "shared/error_wrapper", error_keys: [:places], data_qa: "allocation__places" do %>
+                method: :get,
+                builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
+
+    <%= legend %>
+
+    <%= render "shared/error_wrapper", error_keys: [:number_of_places], data_qa: "allocation__places" do %>
```

Comment:
This shouldn't be needed if we are using the form builder. Just make sure we using the form builders error summary method